dali/operators/python_function/dltensor_function.h

```diff
@@ -258,7 +258,10 @@ class DLTensorPythonFunctionImpl : public Operator<Backend> {
   }
 
   ~DLTensorPythonFunctionImpl() {
-    auto interpreter_lock = py::gil_scoped_acquire();
+    try {
+      auto interpreter_lock = py::gil_scoped_acquire();
+    } catch (...) {
+    }
```

Comment:
Does it make sense to just swallow this exception? What are the chances that we'll ruin something? Perhaps letting the process terminate is exactly what we should do.